commands-v2/ramsete/robotcontainer.py

```diff
@@ -0,0 +1,143 @@
+
+        # Ensures that the max speed is actually obeyed.
+
+        # Apply the previously defined voltage constraint.
+
+        # Start at the origin facing the +x direction.
+        initialPosition = Pose2d(0, 0, Rotation2d(0))
+
+        # Here are the movements we also want to make during this command.
+        # These movements should make an "S" like curve.
+        movements = [Translation2d(1, 1), Translation2d(2, -1)]
+
+        # End at this position, three meters straight ahead of us, facing forward.
+        finalPosition = Pose2d(3, 0, Rotation2d(0))
+
+        # An example trajectory to follow. All of these units are in meters.
+        self.exampleTrajectory = TrajectoryGenerator.generateTrajectory(
+            initialPosition,
+            movements,
+            finalPosition,
+            config,
+        )
+
+        # Below creates the RAMSETE command
+
+        ramseteCommand = RamseteCommand(
+            self.exampleTrajectory,  # The trajectory to follow.
+            self.robotDrive.getPose,  # A reference to a method that will return our position.
+            RamseteController(  # Our RAMSETE controller.
+                constants.kRamseteB, constants.kRamseteZeta
+            ),
+            SimpleMotorFeedforwardMeters(  # A feedforward object for the robot.
+                constants.ksVolts,
+                constants.kvVoltSecondsPerMeter,
+                constants.kaVoltSecondsSquaredPerMeter,
+            ),
+            constants.kDriveKinematics,  # Our drive kinematics.
+            self.robotDrive.getWheelSpeeds,  # A reference to a method which will return a
+            PIDController(
+                constants.kPDriveVel, 0, 0
+            ),  # The turn controller for the left side of the drivetrain.
+            PIDController(
+                constants.kPDriveVel, 0, 0
+            ),  # The turn controller for the right side of the drivetrain.
+            self.robotDrive.tankDriveVolts,  # A reference to a method which will set a specified
+            # voltage to each motor. The command will pass the two parameters.
+            [self.robotDrive],  # The subsystems the command should require.
+        )
+
+        # Reset the robot's position to the starting position of the trajectory.
+        self.robotDrive.resetOdometry(self.exampleTrajectory.initialPose())
+
+        # Return the command to schedule. The "andThen()" will halt the robot after
+        # the command finishes.
+        return ramseteCommand.andThen(self.robotDrive.stopMoving)
+
+    def configureButtons(self):
+        """Configure the buttons for the driver's controller"""
+
+        # We won't do anything with this button itself, so we don't need to
+        # define a variable.
+
+        JoystickButton(
+            self.driverController, XboxController.Button.kBumperRight.value
+        ).whenPressed(self.robotDrive.setSlowMaxOutput).whenReleased(
+            self.robotDrive.setNormalMaxOutput
+        )
```

Comment:
A small stylistic thing: it'd be good to have each method call on its own line, like the upstream examples, for readability. I think Black will accept something like this:
```suggestion
        (
            JoystickButton(
                self.driverController, XboxController.Button.kBumperRight.value
            )
            .whenPressed(self.robotDrive.setSlowMaxOutput)
            .whenReleased(self.robotDrive.setNormalMaxOutput)
        )
```